CONTRIBUTING.md

```diff
@@ -1,5 +1,316 @@
+# Overview
+
+## Technical
```

Comment:
I actually don't think this part is necessary and introduces more maintenance costs...

We're just not writing resumes

Reply:
> I actually don't think this part is necessary and introduces more maintenance costs...
> 
> We're just not writing resumes

All work is being gradually updated in progress. I think the current stage of the SDG needs this, as we need more developers to get involved. Based on the current progress, this part of the maintenance  cost is acceptable. However, it seems that version number information can be removed as it has no significance.

Reply:
Maybe we just need to list a few of them as a reminder of which technologies we are based on? For example, `OpenAI` makes no sense, it's just a client.

Reply:
🤣 Indeed, let me take a look. Delete some unnecessary ones.